### ISSUE TITLE
No Documentation

### ISSUE THREAD

**Issue by mujtaba-ahmad** [liuhong1happy/react-native-wallpaper-manager#1]
Please can you provide documentation on how to use this plugin ?

**Comment by digi0ps** [comment]
+1

**Comment by sidd123456** [comment]
have u used this ?
in my case its not working in android

**Comment by ankita1711** [comment]
is it possible to set lock screen wallpaper?

**Comment by mujtaba-ahmad** [comment]
No i have not used it as its not documented.

**Comment by ankita1711** [comment]
you have any idea how to set lock screen wallpaper using this library  ?

**Comment by SanchezQb** [comment]
Has anyone gotten this library to work?

**Comment by mrf345** [comment]
Yeah i managed, it seems like the npm release is not up to date with the repo.
To get it to work you will have to implement the last pull fix manually by  removing
```java
 @Override
    public List<Class<? extends JavaScriptModule>> createJSModules() {
        return Collections.emptyList();
    }
```
from `WallPaperPackage.java` and make sure to link the package by `react-native link react-native-wallpaper-manager`

**Comment by SanchezQb** [comment]
I tried this. It works in development but doesn't work in the release build

**Comment by mrf345** [comment]
I tried it in release buld .apk and it works just as well. Does it give any errors out ?

**Comment by SanchezQb** [comment]
Nope. No errors. It just doesn't set the wallpaper. I don't know if I did anything wrong while setting it up but if it worked in development, it should work in production right?

**Comment by mrf345** [comment]
Yeah it should, you do development and production test on the same android device ?
What about the callback function response, 
```javascript
let link = 'https://i.redd.it/f0ujrjilatj01.jpg'
WallPaperManager.setWallpaper({uri: link}, (response) => {
   console.log(response)
})
```

if it is successful it should return something similar to this
```javascript
{ url: 'https://i.redd.it/f0ujrjilatj01.jpg',
msg: 'Set Wallpaper Success',
status: 'success' }
```

**Comment by SanchezQb** [comment]
Development and production tests are on the same physical device.  And also, I did get that response

**Comment by mrf345** [comment]
Sorry I'm really out of guesses, have you tried to create a new fresh project just to test the library with ?

**Comment by SanchezQb** [comment]
I haven't tried that yet. I'll do that now. Thank you so much man

**Comment by SanchezQb** [comment]
I just tried it. The error says "Unable to find GlideModule implementation"

**Comment by SanchezQb** [comment]
Works now

**Comment by SanchezQb** [comment]
`def enableProguardInReleaseBuilds = false`. This was previously set to true, that's why it wasn't working in production

**Comment by karanpratapsingh** [comment]
@mrf345 @SanchezQb  Thank you guys it finally worked, haven't tested in production but is working in development 👍

**Comment by SanchezQb** [comment]
Enjoy man 👍🏾
